tests/utils/splitAtom.test.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect, useRef } from 'react'
+
 import { Atom, PrimitiveAtom, atom, useAtom } from 'jotai'
```

Comment:
```suggestion
import { atom, useAtom } from 'jotai'
import type { Atom, PrimitiveAtom } from 'jotai'
```

In jotai, we split js import and type import.

Reply:
Sure, let me fix it.